[14.0][FIX] l10n_es_facturae: obtener certificados con sudo.

LoisRForgeFlow opened the pull request:
De esta forma se permite a usuarios de facturación poder generar
ficheros de factura electrónica firmados.

![image](https://user-images.githubusercontent.com/23449160/170655372-62655314-ae77-4288-951f-4524d167ccae.png)

@ForgeFlow

pedrobaeza commented:
La operación debe hacerse con sudo en su lugar, ya que así se les da acceso para extraer el certificado desencriptado por ejemplo por XML-RPC.

LoisRForgeFlow committed:
[FIX] l10n_es_facturae: obtener certificados con sudo.

De esta forma se permite a usuarios de facturación poder generar
ficheros de factura electrónica firmados.

LoisRForgeFlow commented:
@pedrobaeza cierto, ya está corregido!

pedrobaeza commented:
/ocabot merge patch

Llévalo por favor a la 15
